### DISCUSSION TITLE
can't create app without --quick

### DISCUSSION THREAD

**Discussion by ooeid** [strapi/strapi#7789] [Legacy Topics]
Cannot find module 'C:\Users\DREADK~1\AppData\Local\Temp\strapi9651c38da94a\node_modules\strapi-connector-bookshelf\lib\utils\connectivity.js'

I tried Postgres, MongoDB and MySql but no luck 

what is the problem. 

my node version is v12.18.3

**Comment by derrickmehaffy** [comment]
Unable to reproduce locally on my windows machine, is there any more information you can provide to assist in debugging?

**Reply by ooeid** [reply]
I don't know, I have been trying to fix this for hours. I just reinstalled Node, MongoDB, and even Strapi but no luck. 
any idea why it happens?

**Reply by ooeid** [reply]
> Unable to reproduce locally on my windows machine, is there any more information you can provide to assist in debugging?

I got this when I run React Native 

`Unable to strip library 'C:\Users\DreadKnight\Desktop\code\native-react\speedTypingGame\android\app\build\intermediates\merged_native_libs\debug\out\lib\x86_64\libjscexecutor.so' due to missing strip tool for ABI 'X86_64'. Packaging it as is.`

**Reply by derrickmehaffy** [reply]
@ooeid there isn't much we will be able to do to help you, this looks like an environment issue and I've never seen anything like this. It's not a bug with Strapi itself.

**Reply by ooeid** [reply]
so it is a windows issue? any idea where I should look to debugging?

**Reply by derrickmehaffy** [reply]
It appears to be an issue with your computer, but I'm not sure where to even begin debugging.

A quick google search says this is something with the Android Studio, but the issue doesn't appear to be a bug with Strapi. :confused:
